src/FSharp.Data.GraphQL.Server.AppInfrastructure/GraphQLWebsocketMiddleware.fs

```diff
@@ -285,7 +285,7 @@ type GraphQLWebSocketMiddleware<'Root>(next : RequestDelegate, applicationLifeti
         do! socket |> tryToGracefullyCloseSocketWithDefaultBehavior
       with
       | ex ->
-        logger.LogError (sprintf "Unexpected exception \"%s\" in GraphQLWebsocketMiddleware (handleMessages). More:\n%s" (ex.GetType().Name) (ex.ToString()))
+        logger.LogError("Unexpected exception \"{exceptionname}\" in GraphQLWebsocketMiddleware (handleMessages). More:\n{exceptionstr}", (ex.GetType().Name), ex)
```

Comment:
It will add all the required info including class
```suggestion
        logger.LogError(ex, "Cannot handle WebSocket message")
```

Reply:
Done.